src/frontend/apps/standalone_site/src/components/Modale/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { Box, Button, Layer, LayerProps } from 'grommet';
+import { Nullable } from 'lib-common';
+import { ExitCrossSVG } from 'lib-components';
+import {
+  forwardRef,
+  MutableRefObject,
+  PropsWithChildren,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+const messages = defineMessages({
+  closeButtonTitle: {
+    defaultMessage: 'Close the modale',
+    description: 'Accessibility title for close modale button title.',
+    id: 'components.Modale.closeButtonTitle',
+  },
+});
+
+export interface ModaleApi {
+  open: () => void;
+  close: () => void;
+}
+
+interface ModaleProps extends Omit<LayerProps, 'onClickOutside' | 'onEsc'> {
+  isOpen?: boolean;
+  api?: MutableRefObject<Nullable<ModaleApi>>;
+  onClose?: () => void;
+}
+
+const Modale = forwardRef<
```

Comment:
Can you add a comment explaining what is this component, what is the purpose, etc ?